partner_contact_birthdate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
```

Comment:
The copyright lines that remain should use the word Copyright, not the symbol

Reply:
DONE